cumulusci/robotframework/Salesforce.py

```diff
@@ -166,6 +169,10 @@ def header_field_should_not_have_link(self, label):
         locator = lex_locators["record"]["header"]["field_value_link"].format(label)
         self.selenium.page_should_not_contain_element(locator)
 
+    def click_header_field_link(self, label):
```

Comment:
We should add a test in the ui.robot suite for this keyword

Reply:
Done